src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/Outbound.java

```diff
@@ -0,0 +1,14 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum Outbound implements TransactionType {
+
+    // TO-DO
```

Comment:
"TO-DO" as part of this story? If not, do we know which story will cover it and can indicate it here?

Reply:
I'm unsure about whether this is needed or not, so to simplify things i've converted `TransactionType` to an enum and have removed Outbound/Inbound